builtin/logical/cassandra/path_config_connection.go

```diff
@@ -44,7 +44,7 @@ effect if a CA certificate is provided`,
 			"tls_min_version": &framework.FieldSchema{
 				Type:        framework.TypeString,
 				Default:     "tls12",
-				Description: "Minimum TLS version to use. Accepted values are 'tls10', 'tls11' or 'tls12'. Defaults to 'tls12'",
+				Description: "Minimum TLS version to use. Accepted values are 'tls10', 'tls11', 'tls12' or 'tls13'. Defaults to 'tls12'",
```

Comment:
This back-end is actually deprecated, so I would be in favor of not updating here to give folks more incentive to move over to the database engine. Could we instead add a comment above it that tls13 hasn't been added here because this engine is deprecated, and users should be encouraged to move to the Cassandra database secrets engine?